bin/create.js

```diff
@@ -57,6 +74,10 @@ var tokenOptions = {
         description: 'pre-compile a local file using the indicated library (for now only `babel` is supported and will read `.babelrc` if present)',
         type: 'string',
     },
+    auth: {
+      description: 'set webtask permisions',
```

Comment:
Is this the main switch that controls whether the webtask will require authentication, or something else? If so, the "set webtask permissions" does not express it.